bindings/2.204/GeometryDash.bro

```diff
@@ -6376,16 +6376,16 @@ class GJChallengeItem : cocos2d::CCObject {
 	// virtual ~GJChallengeItem();
 
 	static GJChallengeItem* create();
-	static GJChallengeItem* create(GJChallengeType, int, int, int, gd::string) = win 0x182900;
+	static GJChallengeItem* create(GJChallengeType challengeType, int goal, int reward, int timeLeft, std::string questName) = win 0x182900;
```

Comment:
even though it's std::string in the actual game, it needs to stay gd::string for geode because we are using a different compiler - gd::string is what we use to smooth out the differences

Reply:
It should be fixed now